GravatarApp/Share/ShareViews/ShareHeaderView.swift

```diff
@@ -164,7 +168,8 @@ extension View {
                 geo.safeAreaInsets.top == 0 ?
                     16 : geo.safeAreaInsets.top,
                 imageURL: imageURL,
-                forceRefresh: .constant(false)
+                forceRefresh: .constant(false),
+                windowWidth: .constant(320)
```

Comment:
This value can be taken from the geometry reader instead of a constant:

```suggestion
                windowWidth: .constant(geo.size.width)
```